structure-message/README.md

```diff
@@ -1,6 +1,11 @@
-# SDMX-JSON Structure Message Prototype
+# SDMX-JSON Structure Message
 
-SDMX-JSON Structure Message is a SDMX data exchange format based on JSON.
-Structure Message works together with the SDMX RESTful Web Services API.
-Structure Message focuses on data discovery and it supports all features
-of the SDMX RESTful API for structure queries.
+SDMX-JSON Structure Message is a SDMX data exchange format based on JSON. JSON (JavaScript Object Notation) is a widely used standardised lightweight data-interchange format.
+SDMX-JSON Structure Message works together with the SDMX RESTful Web Services API. Structure Message focuses on data discovery and it supports all features of the SDMX RESTful API for structure queries.
+
+The proposed format conforms to the JSON standard specification, and it supports the SDMX 2.1 Information Model and the full functionality of the SDMX 2.1 RESTful API for structure queries for various types of visualisations. It adheres to the HATEOAS principles. Queries can ask for specific resources (structural SDMX artefacts), and in addition ask for resolving links to and returning related resources.
```

Comment:
`Queries can ask for specific resources (structural SDMX artefacts), and in addition ask for resolving links to and returning related resources.`
This sentence needs some attention. I am not sure I understand the second part.